bigbluebutton-tests/playwright/parameters/customparameters.js

```diff
@@ -230,6 +230,15 @@ class CustomParameters extends MultiUsers {
     expect(await this.modPage.getLocator(e.selectCameraQualityId).inputValue()).toBe('low');
     await this.modPage.waitAndClick(e.startSharingWebcam);
   }
+
+  async webcamBackgroundURL() {
+    await this.modPage.waitAndClick(e.joinVideo);
+    await this.modPage.waitForSelector(e.noneBackgroundButton);
+    await this.modPage.waitAndClick(e.startSharingWebcam);
```

Comment:
before start sharing the webcam, would be good to check if there is the custom virtual background item added here:

![image](https://github.com/user-attachments/assets/9bfefcd4-63f5-49a6-9008-a2af0858bd13)